docs/azure_arc_jumpstart/azure_arc_k8s/aks_iot_edge/aks_iot_edge_terraform/_index.md

```diff
@@ -0,0 +1,232 @@
+
+```shell
+# Manual provisioning with connection string
+[provisioning]
+source = "manual"
+connection_string = "<ADD DEVICE CONNECTION STRING HERE>"
+ ```
+
+* We save the file and execute the following command to synchronize the configuration: 
+
+```shell
+sudo iotedge config apply
+ ```
+
+![Screenshot showing sync config IoT Edge device in VM](./08.png)
+
+* Finally we will return to the Azure Cloud Shell where we will assign to our new device a module to simulate a temperature sensor. For this we will upload the file through the Azure Cloud Shell interface:  
+
+![Screenshot showing sync config IoT Edge device in VM](./09.png)
+
+* Once the file is uploaded, we will launch the following command: 
+
+> **Note: You can see an example of the [deployment.json](https://github.com/Manuss20/Kubernetes_edge/blob/main/deploy/scripts/edge/deployment.json) file that we use.**
+
+```shell
+az iot edge set-modules --hub-name k8sedgejumpstart --device-id "EdgeDeviceSim" --content ./deployment.json
+```
+
+* Now we go to the Azure portal -> K8sEdgeJumpStart and look for our IoT Hub instance -> k8sedgejumpstart: 
+
+![Screenshot showing IoT Edge device in IoT Hub](./10.png)
+
+* By selecting on our IoT Edge device we can see all the information about the modules it is running. If everything has been successful we will see that the "SimulatedTemperatureSensor" module is running correctly.
+
+![Screenshot showing IoT Edge device in IoT Hub](./11.png)
+
+* We can also check from the virtual machine itself the modules that are running at that moment, using the following command: 
+
+```shell
+sudo iotedge list
+```
+
+![Screenshot showing IoT Edge modules running in the device](./12.png)
+
+* Now download the [values.yaml](https://github.com/Azure/iotedge/blob/preview/iiot/kubernetes/charts/edge-kubernetes/values.yaml) file for IoT Edge Helm chart and replace the deviceConnectionString placeholder at the end of the file with the connection string you noted earlier.
+
+```shell
+# Manual provisioning configuration using a connection string
+provisioning:
+  source: "manual"
+  deviceConnectionString: "<ADD DEVICE CONNECTION STRING HERE>"
+  dynamicReprovisioning: false
+```
+
+* We will edit the environment variables section in the included [az_k8sconfig_helm_aks.sh](https://github.com/Manuss20/Kubernetes_edge/blob/main/deploy/scripts/helm/az_k8sconfig_helm_aks.sh) shell script. As in the previous steps we will upload the files to our Azure Cloud Shell.   
+
+![Screenshot showing delete function in Azure Portal](./13.png)
+
+* Once the script run has finished, the AKS cluster will be projected as a new Azure Arc enabled Kubernetes resource. We will proceed to connect to our AKS cluster and in a couple of minutes you should see the workload modules defined in the edge deployment running as pods along with edgeagent and iotedged. 
```

Comment:
Explain why this is happening and how the GitOps flow kicks in. It feels a bit out of context...